rholang/examples/linking/packages/IMapApi.rho

```diff
@@ -36,9 +40,55 @@ export IMapApi in {
               }
             }
           } | 
+          contract GetOrElse(key, map, default, return) = {
```

Comment:
I think the order should be `map, key, default, return`. This would make it consistent with most languages.